test/integration/system_cpu_reader_mock_test.go

```diff
@@ -0,0 +1,15 @@
+package biz
```

Comment:
Nit: What about renaming package to something like `*_test`?

Reply:
Yes, I'll rename, it, it's the legacy name it had from before and now since I already moved it to the test folder, makes sense to change it.

Reply:
I will rename it directly to test, because it's an integration test in it's own package.

Reply:
Done